wasm-node/src/utils.rs

```diff
@@ -0,0 +1,64 @@
+use celestia_node::network;
+use serde_wasm_bindgen::to_value;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+#[derive(PartialEq, Eq, Clone, Copy)]
+pub enum Network {
+    Arabica,
+    Mocha,
+    Private,
+}
+
+#[wasm_bindgen]
+pub fn setup_logging() {
+    console_error_panic_hook::set_once();
+
+    tracing_wasm::set_as_global_default();
+}
+
+#[wasm_bindgen]
+pub fn canonical_network_bootnodes(network: Network) -> JsValue {
```

Comment:
we could not export this function to js and call it from within the `WasmNodeConfig::new`

Reply:
Thanks for the suggestion, I like the idea. I've changed the constructor of `WasmNodeConfig` too. It now takes `Network` only and fills out `bootnodes` and `genesis_hash` with official defaults, obviating the need for `canonical_network_bootnodes` and `network_genesis` to be exported.
If user wants to change the defaults, or connect to private network, they'd use appropriate setters to adjust the config.